playbooks/setup_jenkins_slave.yml

```diff
@@ -2,6 +2,78 @@
 - hosts: job_nodes
   user: root
   vars:
+    ip_address_whitelist: "{{ lookup('env', 'SSH_IP_ADDRESS_WHITELIST') }}"
+  pre_tasks:
+    - name: Install apt packages
+      apt:
+        pkg: "ufw"
+        state: installed
+        update_cache: yes
+
+    - name: Reset UFW
+      ufw:
+        state: reset
+
+    - name: Allow ssh
+      ufw:
+        rule: allow
+        port: 22
+        src: "{{ item }}"
+      with_items: "{{ ip_address_whitelist }}"
+
+    - name: Enable firewall
+      ufw:
+        state: enabled
+        policy: deny
+
+    - name: Add RPC public keys to authorized_keys
```

Comment:
This fails here, because it tries to add a key to the jenkins user that hasn't been created yet.